packages/@uppy/utils/src/UserFacingApiError.js

```diff
@@ -0,0 +1,8 @@
+class UserFacingApiError extends Error {
+  constructor (message) {
+    super(message)
+    this.name = 'UserFacingApiError'
+  }
```

Comment:
```suggestion
  name = 'UserFacingApiError'
```

and maybe add a predicates to make TS use easier

```suggestion
  name = 'UserFacingApiError'
  
  static isUserFacingApiError(err: any): err is UserFacingApiError {
    return err?.name === 'UserFacingApiError'
  }
```